pokemongo_bot/cell_workers/transfer_pokemon.py

```diff
@@ -5,7 +5,13 @@
 from pokemongo_bot.cell_workers.base_task import BaseTask
 
 class TransferPokemon(BaseTask):
+    def initialize(self):
```

Comment:
I think it should only run when the pokemon bag is full or on startup, but I don't think we have a way to check number of pokemon w/o hitting the server, and this was a much faster fix.  Do you have any ideas?